.gitignore

```diff
@@ -1 +1,5 @@
-.idea
\ No newline at end of file
+.idea
+# vim backup files
+*.swp
```

Comment:
`*.sw[pon]` would be shorter

Reply:
I didn't knew you could do that, thank's